extern/sector-storage/ffiwrapper/types.go

```diff
@@ -42,6 +42,7 @@ type Verifier interface {
 }
 
 type SectorProvider interface {
+	// TODO This documentation is probably incorrect.
 	// * returns storiface.ErrSectorNotFound if a requested existing sector doesn't exist
```

Comment:
@magik6k 

```
returns storiface.ErrSectorNotFound if a requested existing sector doesn't exist
```

This is probably wrong because the Remote Sector store implementation fetches the `existing` sector files types from other workers if it does NOT have them locally.

Reply:
Yeah, the doc isn't correct, it probably has been here before we had remote storage support